test/fit_tests/deploy/os_ova_install.py

```diff
@@ -51,64 +56,76 @@ def test01_install_ova_template(self):
-                       "done;" \
-                       "sleep 5;" \
-                       "export vmidstring=`vim-cmd vmsvc/getallvms |grep ora-stack-" + fit_common.ARGS_LIST['stack'] + "`;" \
-                       "for vmid in $vmidstring;" \
-                       "do " \
-                       "vim-cmd vmsvc/power.on $vmid;" \
-                       "exit $?;" \
-                       "done;"
-
-        (command_output, exitstatus) = \
-            fit_common.pexpect.run(
-                            "ssh -q -o StrictHostKeyChecking=no -t " + uname + "@"
-                            + fit_common.ARGS_LIST['hyper'] + " " + esxi_command,
-                            withexitstatus=1,
-                            events={"assword": passwd + "\n"},
-                            timeout=20, logfile=sys.stdout)
-        if exitstatus > 0:
-            print "MAC address processing failed. Exiting..."
-            sys.exit(255)
-
-        # Poll the OVA via ping
-        for dummy in range(0, 30):
-            rc = subprocess.call("ping -c 1 -w 5 " + fit_common.ARGS_LIST['ora'], shell=True)
-            if rc == 0:
-                break
+        for vm in range(0, numvms):
+            print '**** Deploying OVA file on hypervisor ' + fit_common.ARGS_LIST['hyper']
+            rc = subprocess.call("ovftool --X:injectOvfEnv --overwrite "
+                                 "--powerOffTarget --skipManifestCheck -q "
+                                 "--net:'ADMIN'='VM Network' "
+                                 "--net:'CONTROL'='Control Network' "
+                                 "--net:'PDU'='PDU Network' "
+                                 "--name='ora-stack-" + fit_common.ARGS_LIST['stack'] + "-" + str(vm) + "' "
+                                 "--noSSLVerify "
+                                 + ovafile
+                                 + " vi://" + uname + ":" + passwd + "@"
+                                 + fit_common.ARGS_LIST['hyper'], shell=True)
+            # Check for successful completion
+            if rc > 0:
+                print 'OVA installer failed at host: ' + fit_common.ARGS_LIST['hyper'] + "Exiting..."
+                sys.exit(255)
+
+            # Wait for VM to settle
+            fit_common.countdown(30)
+
+            # check number of vms for deployment
+            if numvms == 1:
+                ovamacs = fit_common.STACK_CONFIG[fit_common.ARGS_LIST['stack']]['ovamac']
             else:
-                fit_common.time.sleep(10)
-        self.assertEqual(rc, 0, "VM did not activate.")
-        # Sync time on ORA
-        localdate = fit_common.subprocess.check_output("date +%s", shell=True)
-        fit_common.remote_shell("/bin/date -s @" + localdate.replace("\n", "") + ";/sbin/hwclock --systohc")
+                ovamacs = fit_common.GLOBAL_CONFIG['ovaoui'] + ":00:" + fit_common.ARGS_LIST['stack'] + ":0" + str(vm)
```

Comment:
For both the VM number and the stack number, you can use the string format to enforce fixed with of 2.  For example:

stacknum = 5
print "Stack number {0:02}".format(stacknum)

Will print "Stack number 05"